scripts/qemu-wrapper

```diff
@@ -367,20 +367,22 @@ def main(argv):
     #qemu_args += ["-monitor", "tcp:127.0.0.1:7777,server,nowait"]
 
     if depriv:
+        root_dir = "/var/xen/qemu/{}/".format(domid)
```

Comment:
As Christian commented on the other PR, it would be better to not have a trailing slash here.